pkgs/misc/tmux-plugins/default.nix

```diff
@@ -145,6 +145,24 @@ in rec {
     };
   };
 
+  extrakto = mkDerivation {
+    pluginName = "extrakto";
+    version = "unstable-2021-02-17";
+    src = fetchFromGitHub {
+      owner = "laktak";
+      repo = "extrakto";
+      rev = "45201c7331f9c2964f278df11cf5aba8dc466155";
+      sha256 = "0as8f4l9cqlq0haw95sagv7n83r3m817prjpn4yc5bh8x75adl25";
+    };
+    dependencies = [ pkgs.fzf pkgs.python3 pkgs.xclip ];
+    meta = {
+      homepage = "https://github.com/laktak/extrakto";
+      description = "Fuzzy find your text with fzf instead of selecting it by hand ";
+      license = lib.licenses.mit;
+      platforms = lib.platforms.unix;
```

Comment:
Could you add yourself to the maintainer list [here](https://github.com/NixOS/nixpkgs/blob/master/maintainers/maintainer-list.nix) in its own commit with the message `maintainers: add kidd`.

You can find your `githubId` with `curl -s https://api.github.com/users/kidd | jq '.id'`

Then add yourself as a maintainer of this package.

```nix
maintainers = with lib.maintainers; [ kidd ];
```

Reply:
done :)